Dockerfile

```diff
@@ -2,16 +2,18 @@
 # docker build -t lazygit .
 # docker run -it lazygit:latest /bin/sh
 
-FROM golang:1.14-alpine3.11
+FROM golang:1.18 as build 
```

Comment:
what are your thoughts on using `golang:latest`? That way it doesn't matter if we forget to update the version next time. Go's pretty good with backwards compatibility

Reply:
From the point of *forgetting* it is actually good since you do not have to bother about that. But from the security point of view and also if you consider docker best practices I advise you against the latest tag. Here is a great post about that, please check it out:
* https://platformers.dev/log/2022-03-02-latest-literally-kills-puppies/
I also think that is more **readable** if anyone wants to know about the version that is used.

Reply:
Thanks, I'll give that a read